lib/modules/dosomething/dosomething_reportback/dosomething_reportback.admin.inc

```diff
@@ -239,6 +239,8 @@ function dosomething_reportback_files_form_submit($form, &$form_state) {
     return;
   }
 
+  $updates_to_send_to_rogue = [];
```

Comment:
We might be able to use the new collections functionality that @weerd brought in to phoenix recently. I don't think it is required for merge. But just flagging it.